controllers/useraccount/useraccount_controller.go

```diff
@@ -105,6 +105,13 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		}
 	}
 
+	// retrieve the latest config and use it for this reconciliation
+	config, err := memberCfg.GetConfig(r.Client, namespace)
+	if err != nil {
+		return reconcile.Result{}, errs.Wrapf(err, "unable to get MemberOperatorConfig")
+	}
+	r.config = config
```

Comment:
I have the same concern here

Reply:
Done in 71a0c3e74f849496bfac07f9c3573cef8a80e5e6